packages/quantic/force-app/main/default/lwc/quanticCategoryFacet/quanticCategoryFacet.css

```diff
@@ -19,4 +19,12 @@
 .facet__show-more_text, .facet__show-less_text{
   display: flex;
   align-items: left;
+}
+
+.facet-text__bold {
+  font-weight: 700;
```

Comment:
```
  var(--font-weight-bold, 700);
```

Reply:
On line 14 the color of the show-more button could also leverage the slds design tokens so it adapts to theming.
```
color: var(--brand-text-link, #0176d3);
```
could be a good change to make. Also to make the same change on other facets as well 😅